perllib/FixMyStreet/Cobrand/UKCouncils.pm

```diff
@@ -31,11 +31,6 @@ sub restriction {
     return { cobrand => shift->moniker };
 }
 
-sub problems_restriction {
-    my ($self, $rs) = @_;
-    return $rs->to_body($self->council_id);
-}
```

Comment:
I think it might be confusing to have any report viewable inside a cobrand, even though no links are made, it will be stumbled across/linked to incorrectly or similar. Can we keep the report page behaviour the same, perhaps `admin_problems` should remain `problems`, and instead we 'open up' the list requests only, is that silly?